src/rf/rf/urls.py

```diff
@@ -8,19 +8,13 @@
 from rest_framework import routers
 
 import apps.home.urls
-import apps.home.views
-
+import apps.user.urls
 
 admin.autodiscover()
 
-router = routers.DefaultRouter()
-router.register(r'users', apps.home.views.UserViewSet)
-
 urlpatterns = patterns(
     '',
     url(r'^', include(apps.home.urls)),
-    url(r'^api/', include(router.urls)),
-    url(r'^api-auth/', include('rest_framework.urls',
-                               namespace='rest_framework')),
+    url(r'^auth/', include(apps.user.urls)),
```

Comment:
We typically use a URL prefix that matches the name of the app. So this should by `/user/login` etc.